src/components/common/StaffPayment.vue

```diff
@@ -4,17 +4,88 @@
       <h2>3. Staff Payment</h2>
     </header>
 
-    <div id="staff-payment-container">
-      Under construction!
-    </div>
+    <staff-payment-component
+      :staffPaymentData="getStaffPayment"
+      @update:staffPaymentData="onStaffPaymentData"
+      @valid="staffPaymentFormValid=$event"/>
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
 
-@Component({})
+// Components
+import { StaffPayment as StaffPaymentComponent } from '@bcrs-shared-components/staff-payment'
+
+// Mixins, Interfaces and Enums
+import { FilingTemplateMixin } from '@/mixins'
+import { ActionBindingIF } from '@/interfaces'
+
+import { StaffPaymentIF } from '@bcrs-shared-components/interfaces'
+import { StaffPaymentOptions } from '@bcrs-shared-components/enums'
+
+@Component({
+  components: {
+    StaffPaymentComponent
+  }
+})
 export default class StaffPayment extends Vue {
+  // Global getters
+  @Getter getStaffPayment!: StaffPaymentIF
+
+  // Global setters
+  @Action setStaffPayment!: ActionBindingIF
+
+  private staffPaymentFormValid: boolean = null
```

Comment:
Null? Or false?

Reply:
> Null? Or false?

It shouldn't be a problem as it won't be emitted immediately, but will change it to false since it make more sense

Reply:
Yes please.